app/views/maily/shared/_sidebar.html.erb

```diff
@@ -3,10 +3,20 @@
     <section class="nav_list">
       <h3 class="nav_title"><%= "#{mailer.name.humanize} (#{mailer.total_emails})" %></h3>
       <ul>
+        <% email_name_list = [] %>
         <% mailer.emails_list.each do |email| %>
-          <li><%= link_to email.name.humanize, maily_email_path(mailer: mailer.name, email: email.name), class: sidebar_class(mailer, email) %></li>
+          <% next if email_name_list.include?(email.name) %>
+          <% email_name_list << email.name %>
+          <li>
+            <%= link_to maily_email_path(mailer: mailer.name, email: email.name, version: email.version), class: sidebar_class(mailer, email) do %>
+              <%= email.name.humanize %>
+              <% if email.versions.count > 1 %>
```

Comment:
this statement `email.versions.count > 1` is called couple of times, I think it makes sense to wrap it in a method: `has_versions?`